RealmSwift-swift2.0/Tests/SwiftTestObjects.swift

```diff
@@ -251,11 +251,11 @@ class SwiftIndexedPropertiesObject: Object {
     dynamic var dataCol = NSData()
 
     override class func indexedProperties() -> [String] {
-        return ["stringCol", "intCol", "int8Col", "int16Col", "int32Col", "int64Col", "boolCol", "dateCol"]
+        return ["stringCol", "intCol", "int8Col", "int16Col", "int32Col", "int64Col", "boolCol"]
```

Comment:
I disabled it as it was throwing a LogicError saying that it wasn't supported, but now it works fine so I guess I just had a bug elsewhere.